pkgs/by-name/in/invidious-router/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGo122Module, fetchFromGitLab }:
+let
+  pname = "invidious-router";
+  version = "1.0";
+in buildGo122Module {
+  inherit pname version;
+
+  src = fetchFromGitLab {
+    owner = "gaincoder";
+    repo = pname;
```

```suggestion
    repo = "invidious-router";
```

https://github.com/NixOS/nixpkgs/issues/277994